core/modules/node/node.theme.inc

```diff
@@ -32,6 +32,22 @@ function theme_node_preview_banner_form($variables) {
   return $output;
 }
 
+/**
+ * Returns HTML for node_preview().
+ *
+ * @param $variables
```

Comment:
@docwilmot you'll need to fix this and the other things phpcs picked up.

Reply:
You'll also need to add a `@return array` here I think.

Reply:
Interestingly, none of the other functions in that file have an @return.